README.md

```diff
@@ -148,7 +148,26 @@ client.website.send_message_in_conversation(
 * **Website Visitors**
   * **Count Visitors**: `client.website.count_visitors(website_id)`
   * **List Visitors**: `client.website.list_visitors(website_id, page_number)`
-
+  
+### Plugin
+
+* **Plugin Connect**
+  * **Resolves the current plugin account information**: `client.get_connect_account()`
+  * **Checks whether the connected plugin session is valid**: `client.check_connect_session_validity()`
+  * **Lists all websites linked to connected plugin**: `client.list_all_connect_websites(page_number, filter_configured)`
```

Comment:
Rename descriptions as per https://github.com/crisp-im/node-crisp-api/commit/ae9bfcffd3cbfe2e61aae564844a1c40ab7d1316

Reply:
Addressed comment in https://github.com/crisp-im/python-crisp-api/pull/10/commits/a66079e15946702b57945822036487dbd918524d.